content/blog/2022-03-05_newsletter.md

```diff
@@ -0,0 +1,185 @@
+
+Big news from Movim: [Movim 0.20 "Skiff"](https://mov.im/?node/pubsub.movim.eu/Movim/dd90cd1c-4bcd-45a5-88a3-220fd4280b21) has been released, which features OMEMO support, GUI improvements, and better group chat management.
+
+[Psi+ 1.5.1605](https://sourceforge.net/projects/psiplus/files/Windows/Personal-Builds/KukuRuzo/) has been released with OMEMO fixes.
+
+[Poezio 0.13.2 has been released](https://github.com/poezio/poezio/releases/tag/v0.13.2), this version only fixes the compatibility with the newly released slixmpp 1.8.0, but a much more interesting release is to come soon.
+
+Profanity developers wrote a [blog post](https://profanity-im.github.io/blog/post/profanity-on-pinephone/) on how to run their client on Pinephones.
+
+Tigase released [BeagleIM 5.1](https://github.com/tigase/beagle-im/releases/tag/5.1) and [SiskinIM 7.1](https://github.com/tigase/siskin-im/releases/tag/7.1) featuring improved OMEMO support.
+
+## Servers
+
+[Jackal 0.57.0](https://github.com/ortuman/jackal/releases/tag/v0.57.0), an XMPP server written in Go has been released.
+
+Openfire announces the release of [Openfire 4.7.1](https://discourse.igniterealtime.org/t/openfire-4-7-1-released/91365): Notable fixes include security updates to bundled database drivers, logging configuration fixes, and an important fix for users experiencing troubles with users getting booted from group chat rooms. Furthermore, [Openfire Pàdé 1.6.1](https://discourse.igniterealtime.org/t/openfire-pade-1-6-1-released/91366) has been released. [REST API Openfire plugin 1.7.1](https://discourse.igniterealtime.org/t/rest-api-openfire-plugin-1-7-1-released/91358) hase been released, too.
+
+Snikket published an important update for users of their [server software on ARM devices, including Raspberry Pi boards](https://snikket.org/blog/server-updates-for-arm/).
+
+[Tigase XMPP Server 8.2.0](https://github.com/tigase/tigase-server/releases/tag/tigase-server-8.2.0) has been released.
+
+## Libraries
+
+Mellium Co-op has released [v0.21.0](https://pkg.go.dev/mellium.im/xmpp@v0.21.0) and [v0.21.1](https://pkg.go.dev/mellium.im/xmpp@v0.21.1) of the [melium.im/xmpp](https://mellium.im/xmpp/) library for Go! Highlights for this release include basic support for 
+PubSub and PEP as well as In-Band Bytestreams. More information can be found in the [release announcement](https://opencollective.com/mellium/updates/new-release-mellium-im-xmpp-v0-21-0). Version 0.21.1 fixes a security issue that was discovered in the 
+websocket package and should be preferred when updating. For more information see [CVE-2022-24968](https://mellium.im/cve/cve-2022-24968/). [Mellium Dev Communiqué](https://opencollective.com/mellium/updates/dev-communique-for-february-2022):
```

Comment:
Something got changed here, the stuff at the bottom isn't about Communiqué, it's about the library. Can this be changed back to whatever was in the pad?